.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
   - REMOVE_LOCALES=true
 install:
   - pip install pyyaml coveralls flake8 flake8-import-order doc8
+  - if [[ $TRAVIS_PYTHON_VERSION != 3* ]]; then pip install mock; fi
```

Comment:
Python 2 support and all related code will be dropped soon. So can we avoid this line, and don't execute your new tests if Python < 3? Maybe with something like `@unittest.skipIf(sys.version_info < (3, 0)...` or equivalent?

Reply:
I understand. I don't have strong opinion about that and pushed such changes in c9af954 and 8040230 just in case.

Reply:
:+1: